### PR TITLE
added login page and some css changes

### PR DESCRIPTION
**Pull request by rajutkarsh07**
please make some changes in the app.js so that the navigation bar should not be visible in the login page

### CODE REVIEW

**Comment by rajutkarsh07** [comment]
I think we should remove .component class width from app.css  
we can decide what common width to give to website later....

File: client/src/App.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap");
```
Comment (Shivkant-Chauhan) [inline client/src/App.scss:1]: is this styling for the Sign-in with Google box?

File: client/src/components/login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              }}
+            />
+          </div>
+          <div className="form-btn">
+            <button className="submit-btn">Login</button>
+            <button className="submit-btn submit-btn-blue">
+              <div className="icon-box">
+                <FcGoogle className="icon" />
+              </div>
+              Google Login
```
Comment (Shivkant-Chauhan) [inline client/src/components/login/Login.jsx]:
```suggestion
              Sign-in with Google
```

**Comment by Shivkant-Chauhan** [comment]
> please make some changes in the app.js so that the navigation bar should not be visible in the login page

will work after merging these changes. some routing needs to be changed

**Review by Shivkant-Chauhan — APPROVED** [review]

**Pull request merged**